### DISCUSSION TITLE
Werkzeug 0.11 is vulnerable to CVE-2016-10516, CVE-2019-14322, CVE-2020-28724

### DISCUSSION THREAD

**Discussion by kamalpreetSec** [aquasecurity/trivy#3166] [Q&A]
I need to install Trivy's latest version in Linux_x64.

How can I update Werkzeug to version 2.2.2 in .../integration/testdata/fixtures/fs/pip/requirements.txt as Werkzeug 0.11 is vulnerable to CVEs: CVE-2016-10516, CVE-2019-14322, CVE-2020-28724

**Comment by afdesk** [comment]
@kamalpreetSec it was created specifically. 
`/integration/testdata/` contains data for the integration tests and it has vulnerabilities. 

maybe i miss something, why don't you download Trivy from [releases](https://github.com/aquasecurity/trivy/releases)?
